File: f2f-service/customer.py
import json

class Customer():
    def __init__(self, id, name, gender, customer_dni, customer_channel):
        self.id = id
        self.name = name
        self.gender = gender
        self.customer_dni = customer_dni
        self.customer_channel = customer_channel
    
    def toJSON(self):
        return json.dumps(self, default=lambda o: o.__dict__, 
            sort_keys=True, indent=4)

File: camel-route/src/main/java/ec/insuasti/ups/homeworks/camelroute/Customer.java
package ec.insuasti.ups.homeworks.camelroute;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@ToString
@NoArgsConstructor
public class Customer {
    private int id;
    private String name;
    private String gender;
    private String customer_dni;
    private String customer_channel;

}
